src/Validator/Constraints/PasswordStrengthValidator.php

```diff
@@ -139,7 +139,6 @@ public function validate($password, Constraint $constraint)
 
         if ($passwordStrength < $constraint->minStrength) {
             $parameters = array(
-                '{{ length }}' => $constraint->minLength,
```

Comment:
This is a BC break, as people could rely on it in custom messages (even though it is not really useless)